atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampStore.java

```diff
@@ -112,33 +143,54 @@ public synchronized void backupExistingTimestamp() {
      * This may be implemented following the inverse of the backup process:
      *  - Read the value of the timestamp. If this is readable, skip.
      *  - Read the value of the backup (BT). If this is unreadable, fail.
-     *  - Do a conditional logged batch:
```

Comment:
Yep, I don't think the comments explaining the implementation are necessary. 

You're right - I've now added this step. I guess it was a good thing that I wrote these comments out when thinking of how to structure the process.